docs/requirements.txt

```diff
@@ -0,0 +1,3 @@
+numpy==1.19.2
```

Comment:
Does this file have a different purpose than `reqs/docs.pip`? We probably don't want both. I'm not sure, but you probably just wan to override that file.

Are there not other requirements for generating the docs, such as sphinx?

Reply:
This is for the Read-the-Docs host, which I was using as a preview. Not needed anymore so I will remove it from the next iteration.